CHANGES.rst

```diff
@@ -8,6 +8,12 @@ New Features
 
 - ``astropy.constants``
 
+  - Constants are now organized into version classes, with updated physical
+    constants in the CODATA2014 class and astronomical constants in the
+    ASTRON_V2_0 class. These updated constants are available in the
+    ``astropy.constants`` namespace. The constants for astropy v1.3
+    are retained in separate version classes. [#4958, #4109, #4397]
```

Comment:
Please add the number of this PR here. Also remove 4109 and 4397 as those PRs were closed without merge and you didn't bring over the code here.